pkg/datapath/loader/netlink.go

```diff
@@ -277,22 +277,28 @@ func setupVxlanDevice(logger *slog.Logger, sysctl sysctl.Sysctl, port, srcPortLo
 		PortHigh:  int(srcPortHigh),
 	}
 
+	// It's possible to create multiple vxlan devices with the same dstport,
+	// though only one of them can be 'up'. Delete an existing vxlan device
+	// with a mismatching port before attempting to create a new one to
+	// to avoid ensureDevice setting up the old interface. This avoids the
```

Comment:
I made a typo in the initial comment:

```suggestion
	// avoid ensureDevice setting up the old interface. This avoids the
```

Reply:
Thanks, done